spec/spec_helper.rb

```diff
@@ -16,7 +16,7 @@
 Dir[File.join(File.dirname(__FILE__), 'support/**/*.rb')].each { |f| require f }
 
 # Requires factories defined in lib/solidus_log_viewer/factories.rb
```

Comment:
Do we need to update this comment as well?

Reply:
I believe the makes perfect sense!